tests/acceptance/run_test.go

```diff
@@ -0,0 +1,66 @@
+package acceptance
+
+import (
+	"os/exec"
+	"strings"
+	"testing"
+)
+
+func TestRunCommandHelp(t *testing.T) {
+	// Test run command help
+	cmd := exec.Command("kodelet", "run", "--help")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("Failed to execute run --help: %v", err)
+	}
+
+	outputStr := strings.TrimSpace(string(output))
+
+	// Should contain usage information
+	if !strings.Contains(outputStr, "Usage") && !strings.Contains(outputStr, "usage") {
+		t.Errorf("Help output should contain usage information: %s", outputStr)
+	}
+
+	// Should contain run-specific flags
+	if !strings.Contains(outputStr, "--no-save") && !strings.Contains(outputStr, "--follow") {
+		t.Errorf("Help output should contain run-specific flags: %s", outputStr)
+	}
+}
+
```

Comment:
add a few core test cases:

1. create a hello.txt with "hello world" as the content
2. ask "is the operating system linux or windows", the answer is linux (with case insenstive check)
3. write a fibonacci program in `$testdir/fib.py` then verify the fib implementation


ideally implement this in a table test manner.